realsense2_camera/launch/rs_launch.py

```diff
@@ -58,6 +58,7 @@
                            {'name': 'initial_reset',                'default': 'false', 'description': "''"},                           
                            {'name': 'allow_no_texture_points',      'default': 'false', 'description': "''"},                           
                            {'name': 'pointcloud.ordered_pc',        'default': 'false', 'description': ''},
+                           {'name': 'publish_tf',                   'default': 'true', 'description': 'enable/disable publishing tf'},
```

Comment:
it is preferred to explain more in the description field, specially for these two parameters since they can confuse users.
- publish_tf: [bool] enable/disable publishing static or dynamic TF
- tf_publish_rate: [double] rate in HZ for publishing dynamic TF


It is also preferred to add these and elaborate more in the README.md file
1- to explain that publish_tf:=false will not publish any static/dynamic TFs, even if tf_publish_rate is bigger than 0.0
2- to explain that publish_tf:=true, will automatically publish static TFs. If dynamic TFs are needed, user should set the tf_publish_rate for it, and it should be > 0.0